icd/api/vk_descriptor_set.cpp

```diff
@@ -89,6 +89,27 @@ void DescriptorSet<numPalDevices>::Reassign(
 
 }
 
+// =====================================================================================================================
+// Writes the immutable samplers in the layout to memory.
+template <uint32_t numPalDevices>
+void DescriptorSet<numPalDevices>::WriteImmutableSamplers()
+{
+    for (uint32_t deviceIdx = 0; deviceIdx < numPalDevices; deviceIdx++)
+    {
+        for (uint32_t bindingIndex = 0; bindingIndex < Layout()->Info().count; ++bindingIndex)
+        {
+            const DescriptorSetLayout::BindingInfo& bindingInfo = Layout()->Binding(bindingIndex);
+
+            if (bindingInfo.imm.dwSize != 0)
+            {
+                uint32_t* pSamplerDesc =(Layout()->Info().imm.pImmutableSamplerData + bindingInfo.imm.dwOffset);
```

Comment:
Hi @s-perron  please remove the extra parentheses

Reply:
please add a space between "=" and "Layout"

Reply:
Done.